content/docs/dashboard/features.md

```diff
@@ -82,3 +143,8 @@ Currently, mails are sent in the following scenarios:
 - And to reset the password of a user.
 
 If nothing is configured, no mails are sent and the admin must reset the password for the user. 
+
+![](/smtp.gif)
+
+
+**Important:** This feater is implemented in the frontend but not yet 100% in the backend and therefore might not work yet.
```

Comment:
Is there an issue for it? If no, can we create one and add a link to the issue? That ways once we overhaul we can close the issue and remember to update the wiki as well.

Reply:
Found the open issue: https://github.com/oakestra/oakestra/issues/182